spirv_cross_c.h

```diff
@@ -745,6 +745,7 @@ typedef enum spvc_compiler_option
 	SPVC_COMPILER_OPTION_MSL_FORCE_FRAGMENT_WITH_SIDE_EFFECTS_EXECUTION = 89 | SPVC_COMPILER_OPTION_MSL_BIT,
 
 	SPVC_COMPILER_OPTION_HLSL_USE_ENTRY_POINT_NAME = 90 | SPVC_COMPILER_OPTION_HLSL_BIT,
+	SPVC_COMPILER_OPTION_HLSL_PRESERVE_STRUCTURED_BUFFERS = 91 | SPVC_COMPILER_OPTION_HLSL_BIT,
```

Comment:
Same concern as last PR. Need to bump the defines for minor version.

Reply:
Bumped the API to 63 in the other PR, so #2373 must go in before this one

Otherwise fixed in [4b98efb](https://github.com/KhronosGroup/SPIRV-Cross/pull/2374/commits/4b98efbf7f5bbfa61dd4c3a0e3aaab43036e471c)